sklearn/metrics/_classification.py

```diff
@@ -769,12 +798,27 @@ def jaccard_score(
     ...                    [1, 1, 0]])
     >>> y_pred = np.array([[1, 1, 1],
     ...                    [1, 0, 0]])
+    >>> y_true_with_empty = np.array(
+    ...                   [[0, 1, 1],
+    ...                    [0, 0, 0],
+    ...                    [1, 1, 0]])
+    >>> y_pred_with_empty = np.array(
+    ...                   [[1, 1, 1],
+    ...                    [0, 0, 0],
+    ...                    [1, 0, 0]])
 
     In the binary case:
 
     >>> jaccard_score(y_true[0], y_pred[0])
     0.6666...
 
+    >>> jaccard_score(y_true_with_empty[1], y_pred_with_empty[1])
```

Comment:
We only use the row indexed 1 in `y_true_with_empty` and `y_pred_with_empty`. I think it will be more obvious to hard code the zero:

```suggestion
    >>> jaccard_score([0, 0, 0], [0, 0, 0])
```

Also I prefer to append this to the end of the Example section. This way, the example flows naturally with the original `y_true` and `y_pred`.